src/bin-utils.js

```diff
@@ -31,6 +31,37 @@ const formatAsciiString = function(e) {
 };
 
 /**
+ * Creates an object for sending to a web worker modifying properties that are TypedArrays
+ * into seperated properties for the buffer, byteOffset, and byteLength.
+ *
+ * @param {Object} message
+ *        Object of properties and values to send to the web worker
+ * @return {Object}
+ *         Modified message with TypedArray values expanded
+ * @function transferableMessage
+ */
+const transferableMessage = function(message) {
```

Comment:
`Transferrable` is the name given to the class of objects that can be zero-copied into a webworker. ArrayBuffers are one of those types but to be able to reconstruct them we need some more data. This function prepares them for transferrability. Personally, I think the name mostly makes sense but I think it might be better to call it `createTransferableMessage`.